src/test/java/uk/gov/pay/api/json/CreateCardPaymentRequestDeserializerTest.java

```diff
@@ -145,6 +145,46 @@ public void deserialize_shouldDeserializeARequestWithDelayedCaptureEqualsFalseSu
         assertThat(paymentRequest.getDelayedCapture(), is(Optional.of(Boolean.FALSE)));
     }
 
+    @Test
+    public void deserialize_shouldDeserializeARequestWithMotoEqualsTrueSuccessfully() throws Exception {
+        // language=JSON
+        String validJson = "{\n" +
+                "  \"amount\": 27432,\n" +
+                "  \"reference\": \"Some reference\",\n" +
+                "  \"description\": \"Some description\",\n" +
+                "  \"return_url\": \"https://somewhere.gov.uk/rainbow/1\",\n" +
+                "  \"moto\": true\n" +
+                "}";
+
+        CreateCardPaymentRequest paymentRequest = deserializer.deserialize(jsonFactory.createParser(validJson), ctx);
+
+        assertThat(paymentRequest.getAmount(), is(27432));
+        assertThat(paymentRequest.getReference(), is("Some reference"));
+        assertThat(paymentRequest.getDescription(), is("Some description"));
+        assertThat(paymentRequest.getReturnUrl(), is("https://somewhere.gov.uk/rainbow/1"));
+        assertThat(paymentRequest.getMoto(), is(Optional.of(Boolean.TRUE)));
+    }
+
+    @Test
+    public void deserialize_shouldDeserializeARequestWithMotoEqualsFalseSuccessfully() throws Exception {
+        // language=JSON
+        String validJson = "{\n" +
+                "  \"amount\": 27432,\n" +
+                "  \"reference\": \"Some reference\",\n" +
+                "  \"description\": \"Some description\",\n" +
+                "  \"return_url\": \"https://somewhere.gov.uk/rainbow/1\",\n" +
+                "  \"moto\": false\n" +
+                "}";
+
+        CreateCardPaymentRequest paymentRequest = deserializer.deserialize(jsonFactory.createParser(validJson), ctx);
+
+        assertThat(paymentRequest.getAmount(), is(27432));
+        assertThat(paymentRequest.getReference(), is("Some reference"));
+        assertThat(paymentRequest.getDescription(), is("Some description"));
+        assertThat(paymentRequest.getReturnUrl(), is("https://somewhere.gov.uk/rainbow/1"));
+        assertThat(paymentRequest.getMoto(), is(Optional.of(Boolean.FALSE)));
+    }
+
```

Comment:
For all the new tests added above, consider using Parameters, e.g. at https://github.com/alphagov/pay-connector/blob/master/src/test/java/uk/gov/pay/connector/charge/service/ChargeCancelServiceTest.java#L88-L92, in another PR.